examples/all-clusters-app/all-clusters-common/all-clusters-app.matter

```diff
@@ -7969,7 +7969,7 @@ endpoint 1 {
     callback attribute eventList;
     callback attribute attributeList;
     ram      attribute featureMap default = 1;
-    ram      attribute clusterRevision default = 5;
```

Comment:
This changes per-endpoint default however this seems tedious to keep track off.

I see that the scenesmanagement is still revision 5 as a default on line 3727 ... could you change the XML definitions to bump up the revision rather than just changing the zap files?

Reply:
`src/app/zap-templates/zcl/data-model/chip/scene.xml` should be modified to set the global attribute to 6